pkg/oidcclient/login.go

```diff
@@ -622,6 +627,7 @@ func (h *handlerState) promptForWebLogin(ctx context.Context, authorizeURL strin
 		token, err := h.redeemAuthCode(ctx, code)
 		h.callbacks <- callbackResult{token: token, err: err}
 	}()
+	return wg.Wait
 }
 
 func promptForValue(ctx context.Context, promptLabel string) (string, error) {
```

Comment:
In the context of an LDAP login, should this function get called with context.Background()? Otherwise, doesn't this mean that LDAP username prompts will eventually time out and get cancelled? If so, I don't think that was intended.

Reply:
Maybe this is okay even though it is a change. It seems like the timeout is set to 90 minutes, right? If you didn't type your password after 90 minutes then you're probably never going to type it. :laughing